Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM rust:1.59 as build
```

Comment:
`rust:1-whatever version of ubuntu you're using afterwards`
(For example just `rust:1-ubuntu`)
We dont really care about strict rust version, so we're allow docker to update to newer rust versions

Reply:
Didn`t find rust ubuntu docker image. As I understand, official rust image is based on Debian, so I set bullseye version. Also set architecture amd64 (but not sure if it needed) and made image slim.